pkg/api/component_report.go

```diff
@@ -1188,18 +1194,27 @@ func (c *componentReportGenerator) assessComponentStatus(sampleTotal, sampleSucc
 			if c.MinimumFailure != 0 && (sampleTotal-sampleSuccess-sampleFlake) < c.MinimumFailure {
 				return apitype.NotSignificant, fischerExact
 			}
+
 			basisPassPercentage := float64(baseSuccess+baseFlake) / float64(baseTotal)
 			samplePassPercentage := float64(sampleSuccess+sampleFlake) / float64(sampleTotal)
+			effectivePityFactor := c.PityFactor
+			if approvedRegression != nil && approvedRegression.RegressedPassPercentage < int(basisPassPercentage) {
+				// product owner chose a required pass percentage, so we all pity to cover that approved pass percent
+				// plus the existing pity factor to limit, "well, it's just *barely* lower" arguments.
+				effectivePityFactor = int(basisPassPercentage) - approvedRegression.RegressedPassPercentage + c.PityFactor
```

Comment:
Maybe also a sanity check to make suer effectivePityFactor is never below 0 or c.PityFactor.  Could just log and reset, or panic since it probably means the approvedRegression has messed up values